js/src/ui/AccountCard/accountCard.css

```diff
@@ -0,0 +1,105 @@
+/* Copyright 2015, 2016 Parity Technologies (UK) Ltd.
+/* This file is part of Parity.
+/*
+/* Parity is free software: you can redistribute it and/or modify
+/* it under the terms of the GNU General Public License as published by
+/* the Free Software Foundation, either version 3 of the License, or
+/* (at your option) any later version.
+/*
+/* Parity is distributed in the hope that it will be useful,
+/* but WITHOUT ANY WARRANTY; without even the implied warranty of
+/* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+/* GNU General Public License for more details.
+/*
+/* You should have received a copy of the GNU General Public License
+/* along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+.account {
+  padding: 1em;
+  margin: 0.5em 0;
+
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+
+  background-color: rgba(255, 255, 255, 0.25);
```

Comment:
Find this not matching with what we have everywhere else. (Also makes it difficult to read since it ends up being a gray blur - especially with the tags)